File: PY/2061.py
# BEE 2016 - As Abas de Péricles - Python

def abas(n, acoes):
    for acao in acoes:
        if (acao == 'fechou'):
            n += 1
        else:
            n -= 1
    return n


n, n_acoes = input().split()
acoes = [input() for i in range(int(n_acoes))]
print(abas(int(n), acoes))

File: C/2061.c
#include <stdio.h>
#include <string.h>

int abas(int n, int m)
{
    char acao[7];
    for (int i = 0; i < m; i++)
    {
        fflush(stdin);
        gets(acao);
        strcmp(acao, "fechou") == 0 ? n++ : n--;
    }
    return n;
}

int main()
{
    int n, m;
    scanf("%d %d", &n, &m);
    printf("%d\n", abas(n, m));
    return 0;
}